### PR TITLE
CSRF mail less disturbing

### PR DESCRIPTION
**Pull request by SansPseudoFix**
Changes based on the conversation on #7193. I kept @goobertron text [here](https://github.com/diaspora/diaspora/issues/7193#issuecomment-263902310) and changed "account" by "session" as mentioned [here](https://github.com/diaspora/diaspora/issues/7193#issuecomment-263949350) by @SuperTux88

### CODE REVIEW

File: config/locales/diaspora/en.yml [hunk 1]
```diff
@@ -715,12 +715,23 @@ en:
...
+          - Another website making requests, with or without your permission;
+          - Various other external tools;
+          - Malicious code trying to access your data.
 
         For more information on CSRF see [%{link}](%{link}).
 
-        Sorry,
+        Don’t worry; you can safely sign in again now.
+
+        If you see this message regularly, please check your browsing settings.
```
Comment (SuperTux88) [inline config/locales/diaspora/en.yml]: > If you see this message repeatedly, please check the points above, including any browser add-ons.

As mentioned here: https://github.com/diaspora/diaspora/issues/7193#issuecomment-263936222

**Review by SuperTux88 — CHANGES_REQUESTED** [review]

File: config/locales/diaspora/en.yml (hunk 1 above)
Comment (SuperTux88) [inline config/locales/diaspora/en.yml]: Changed it myself and fixed the broken spec.

**Review by SuperTux88 — APPROVED** [review]

File: config/locales/diaspora/en.yml (hunk 1 above)
Comment (SansPseudoFix) [inline config/locales/diaspora/en.yml]: Sorry for missing regularly -> repeatedly. I was planning to do it tomorrow but it's cool, thx!

Comment (SuperTux88) [inline config/locales/diaspora/en.yml]: No worries, we are planning to freeze tonight, so I thought I do it myself, since it's a quick fix :)

**Comment by denschub** [comment]
Merged as 6c37171f9f37edcb79770c1c37c609d692f0d4b4 61300dab592537d846c1c05d273841752e4551b5, gracias.

**Comment by Flaburgan** [comment]
Thanks @SansPseudoFix and thanks for the merge. Would it be possible to push the strings to webtranslateit so the email will be localized in the 0.6.2 release?

**Comment by denschub** [comment]
> Would it be possible to push the strings to webtranslateit

yes, sure! Thanks for the reminder, pushed it now.

**Pull request closed without merging**